ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.example.comfortzone.notification;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.widget.CompoundButton;
+import android.widget.TextView;
+
+import com.example.comfortzone.R;
+import com.google.android.material.switchmaterial.SwitchMaterial;
+
+public class NotificationActivity extends AppCompatActivity {
+
+    private TextView tvNotificationTime;
+    private SwitchMaterial swNotification;
+    private SharedPreferences sharedPref;
+    public static final String KEY_IS_NOTIFICATION = "isNotification";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_notification);
+
+        sharedPref = getSharedPreferences(getString(R.string.key_shared_pref_activity), MODE_PRIVATE);
```

Comment:
we dont' have to store the sharedpreferences as it may change, just get it whenever you use it. If you need it in populateViews, you can pass it in or just get it from there